cli/ballerina-cli/src/main/resources/cli-help/ballerina-profile.help

```diff
@@ -0,0 +1,27 @@
+NAME
+       ballerina-profile - Run Ballerina profiler on the source and generate flame graph
+
+SYNOPSIS
+       bal profile [OPTIONS] [<package>|<source-file>]
+
+
+DESCRIPTION
+       Compile the current package and run the program with ballerina profiler. This command generates an html file
```

Comment:
```suggestion
       Compile the current package and run the program with Ballerina Profiler. This command generates an html file
```
Like `NetBeans Profiler` or `IntelliJ Profiler` I think we can call this `Ballerina Profiler`.